composer.json

```diff
@@ -16,20 +16,19 @@
     }
   ],
   "require": {
-    "php": "^7.0",
+    "php": "^7.2",
     "ext-mysqli": "*",
-    "typo3/cms-core": "^8.7 || ^9.5 || ^10",
```

Comment:
^10 is removed as the QueryBuilder->set() isn't compliant with the 10.4 interface. So need a fix before being TYPO3 10LTS compatible.